app/routes/user/permissions.tsx

```diff
@@ -0,0 +1,300 @@
+              type="submit"
+              disabled={disabled}
+              name="intent"
+              value="approved"
+            >
+              Approve
+            </Button>
+          </approvalFetcher.Form>
+          <rejectFetcher.Form method="post">
+            <input
+              type="hidden"
+              name="requestorSub"
+              value={endorsementRequest.requestorSub}
+            />
+            <Button
+              type="submit"
+              outline
+              disabled={disabled}
+              name="intent"
+              value="rejected"
+            >
+              Reject
+            </Button>
+          </rejectFetcher.Form>
+          <reportFetcher.Form method="post">
+            <input
+              type="hidden"
+              name="requestorSub"
+              value={endorsementRequest.requestorSub}
+            />
+            <Button
+              type="submit"
+              secondary
+              disabled={disabled}
+              name="intent"
+              value="reported"
+            >
+              Reject and Report
+            </Button>
+          </reportFetcher.Form>
+        </ButtonGroup>
+      </Grid>
+    )
+  } else {
+    return (
+      <Grid row style={disabled ? { opacity: '50%' } : undefined}>
+        <div className="tablet:grid-col flex-fill">
+          <div className="margin-y-0">
+            <strong>Requested Endorser: </strong>
+            {endorsementRequest.endorserEmail}
+          </div>
+          <div className="margin-y-0">
+            <strong>Status: </strong>
+            {endorsementRequest.status}
+          </div>
+        </div>
+        <ButtonGroup className="tablet:grid-col flex-auto flex-align-center">
+          <reportFetcher.Form method="post">
+            <input
+              type="hidden"
+              name="endorserSub"
+              value={endorsementRequest.endorserSub}
+            />
+            <Button
+              type="submit"
+              secondary
+              disabled={disabled}
+              name="intent"
+              value="delete"
+            >
+              Delete
+            </Button>
+          </reportFetcher.Form>
+        </ButtonGroup>
+      </Grid>
+    )
+  }
+}
+
+export function EndorsementRequestForm() {
+  const { validUsers } = useLoaderData<typeof loader>()
+  const [endorserSub, setEndorserSub] = useState('')
+
+  return (
+    <Form method="post">
+      <h2 id="modal-request-heading">Request Endorsement</h2>
+      <div className="usa-prose">
+        Requesting an endorsement from another user will share your email with
+        that individual. Please keep this in mind when submitting. Enter the
+        email of the individual you want to be endorsed by. They will receive a
+        notification alerting them to this request.
+      </div>
+      <Label htmlFor="endorserSub">Endorser</Label>
+      <input type="hidden" name="endorserSub" value={endorserSub} />
```

Comment:
Is the hidden field necessary? Does the ComboBox not populate anything in the form data?

Reply:
Yea, another one where thats how I thought it should work too, but I found that as I tried the form it wasn't updating the value. The ComboBox has a required onChange, but not a value property